RFC Draft.md

```diff
@@ -377,9 +377,368 @@ const uploadedObjectReference = await uploadTask.result;
 
 Try out the proposed `storage` types here: https://stackblitz.com/edit/rfc-typescript-v6?file=examples-storage.ts
 
-# `API` Category Changes
+# GraphQL `API` Category Changes
 
-Amplify is proposing the following changes for the `API` category.
+Amplify is proposing the following changes for the GraphQL `API` category to improve type safety and readability.
+
+## Split GraphQL operations into `query()`, `mutation()`, and `subscription()` APIs
+
+To better capture customer intent and simplify API types we will split up the `graphql` API into individual APIs for queries, mutations, and subscriptions.
```

Comment:
```suggestion
To better capture customer intent and simplify API types we will split up the `graphql` API into individual APIs for queries, mutations, and subscriptions. We're going to retain the `graphql()` operation in case you want to issue multiple queries/mutations in a single request.
```